### PR TITLE
Set permalink for confirmation page

### PR DESCRIPTION
**Pull request by adborden**
This allows you to use `/confirmation/` for the url path.

### CODE REVIEW

File: prototypes/form-wizard/confirmation.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 layout: page
+permalink: /confirmation/
```
Comment (pkarman) [inline prototypes/form-wizard/confirmation.html:3]: FYI, I had removed these to keep them out of the nav bar, where the "Thank you for your submission" shows up automatically.

File: prototypes/form-wizard/confirmation.html [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+title: Thank you for your submission
```
Comment (pkarman) [inline prototypes/form-wizard/confirmation.html]: some experimentation locally shows that omitting the `title` but keeping the `permalink` is the compromise we want. The `title` puts it in the nav bar, which we don't want.

Comment (adborden) [reply prototypes/form-wizard/confirmation.html]: Given the feedback we've heard, maybe we want to get rid of the nav all together.

[Navbar code is here](https://github.com/18F/foia-recommendations/blob/master/prototypes/form-wizard/_includes/header.html#L30), you can see it's conditioned on title, we can add a property like `show_in_nav` to the frontmatter and condition off that for the nav.

**Review by pkarman — APPROVED** [review]
lgtm

**Pull request merged**